enum_list.rb

```diff
@@ -0,0 +1,25 @@
+require_relative './enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ } You should implement your own #each method that yields all members of `@list` in succession rather than using the ruby built-in #each method. Let's fix that

Reply:
Plaese help us understand how it should be do we thought this is an implementation of #each method

```
    class ArrayWrapper
    
      include CustomEnumerable
    
      def initialize(*items)
        @items = items.flatten
      end
    
      def each(&block)
        @items.each(&block)
        self
      end
    
      def ==(other)
        @items == other
      end
    
    end
```

[Here ](https://mauricio.github.io/2015/01/12/implementing-enumerable-in-ruby.html)is the reference.